Apps/Sandcastle/gallery/Elevation Band Material.html

```diff
@@ -0,0 +1,325 @@
+    />
+    <meta
+      name="description"
+      content="Apply elevation band material to the globe."
+    />
+    <meta name="cesium-sandcastle-labels" content="Showcases" />
+    <title>Cesium Demo</title>
+    <script type="text/javascript" src="../Sandcastle-header.js"></script>
+    <script
+      type="text/javascript"
+      src="../../../Build/CesiumUnminified/Cesium.js"
+      nomodule
+    ></script>
+    <script type="module" src="../load-cesium-es6.js"></script>
+  </head>
+  <body
+    class="sandcastle-loading"
+    data-sandcastle-bucket="bucket-requirejs.html"
+  >
+    <style>
+      @import url(../templates/bucket.css);
+    </style>
+    <div id="cesiumContainer" class="fullSize"></div>
+    <div id="loadingOverlay"><h1>Loading...</h1></div>
+    <div id="toolbar">
+      <table>
+        <tbody>
+          <tr>
+            <td>Background Transparency</td>
+            <td>
+              <input
+                type="range"
+                min="0.0"
+                max="1.0"
+                step="0.01"
+                data-bind="value: backgroundTransparency, valueUpdate: 'input'"
+              />
+            </td>
+          </tr>
+          <tr>
+            <td>Band Transparency</td>
+            <td>
+              <input
+                type="range"
+                min="0.0"
+                max="1.0"
+                step="0.01"
+                data-bind="value: bandTransparency, valueUpdate: 'input'"
+              />
+            </td>
+          </tr>
+          <tr>
+            <td>Band Thickness</td>
+            <td>
+              <input
+                type="range"
+                min="10"
+                max="1000"
+                step="1"
+                data-bind="value: bandThickness, valueUpdate: 'input'"
+              />
+            </td>
+          </tr>
+          <tr>
+            <td>Band 1 Position</td>
+            <td>
+              <input
+                type="range"
+                min="4000"
+                max="8848"
+                step="1"
+                data-bind="value: band1Position, valueUpdate: 'input'"
+              />
+            </td>
+          </tr>
+          <tr>
+            <td>Band 2 Position</td>
+            <td>
+              <input
+                type="range"
+                min="4000"
+                max="8848"
+                step="1"
+                data-bind="value: band2Position, valueUpdate: 'input'"
+              />
+            </td>
+          </tr>
+          <tr>
+            <td>Band 3 Position</td>
+            <td>
+              <input
+                type="range"
+                min="4000"
+                max="8848"
+                step="1"
+                data-bind="value: band3Position, valueUpdate: 'input'"
+              />
+            </td>
+          </tr>
+          <tr>
+            <td>Gradient</td>
+            <td>
+              <input type="checkbox" data-bind="checked: gradient" />
```

Comment:
The logic for this seems to be flipped. When it's unchecked I see a gradient. When it's checked the gradient disappears.